File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        int ns = s.length(), nt = t.length();
        if(ns != nt) return false;
        unordered_map<char,int> mp;
        for(int i = 0;i<ns;i++){
             mp[s[i]]++;
            mp[t[i]]--;
        }
        for(auto it : mp) if(it.second != 0) return false;
        return true;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.cs
public class Solution {
    public bool IsAnagram(string s, string t) {
        int[] count = new int[26];
        for(int i = 0; i < s.Length; i++) count[s[i] - 'a']++;
        for(int i = 0; i < t.Length; i++) count[t[i] - 'a']--;
        for(int i = 0; i < count.Length; i++) if(count[i] != 0) return false;
        return true;
    }
}